build-system/tasks/cherry-pick.js

```diff
@@ -156,11 +156,13 @@ function cherryPick() {
       `Cherry-picked ${commits.length} commits onto release ${onto}`
     );
     process.exitCode = 0;
+    return Promise.resolve();
```

Comment:
Pretty certain `process.exitCode = 0` can be deleted, since that's the default value.

Reply:
Done